classes/locations.php

```diff
@@ -335,29 +335,29 @@ function additional_information( $post ) {
 
 					<!-- Phone -->
 					<tr valign="top">
-						<td width="150"><label for="location_phone"><?php _e( 'Phone', 'simplemap' ); ?></label></td>
+						<td width="150"><label for="location_phone"><?php _e( 'Phone', 'SimpleMap' ); ?></label></td>
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed
* Space after opening parenthesis of function call prohibited
* Expected 0 spaces before closing bracket; 1 found

<!-- Comment by [SideCI](https://sideci.com) -->